data/saas/config/braze_config.yml

```diff
@@ -0,0 +1,106 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Braze SaaS Config
+  type: braze
+  description: A sample schema representing the braze connector for Fidesops
+  version: 0.0.1
+
+  connector_params:
+    - name: domain
+    - name: api_key
+    - name: identity_email
+    - name: subscription_group_id
+
+  client_config:
+    protocol: https
+    host: <domain>
+    authentication:
+      strategy: bearer
+      configuration:
+        token: <api_key>
+
+  test_request:
+    method: GET
+    path: /email/hard_bounces/
+    headers:
```

Comment:
The test request will use the same auth as the `client_config` so there is no need to specify the headers here. Also, you can just use the `<api_key>` placeholder here without having to use a `param_value` (I know this last one is a bit weird).